### PR TITLE
added new milestone types

### PR DESCRIPTION
**Pull request by jc-umana**
Defined ALLOWED_MILESTONE_TYPES and put them in types field

### CODE REVIEW

**Comment by jc-umana** [comment]
I've tried... and it would seem that I have failed.

File: regolith/schemas.py [hunk 1]
```diff
@@ -1168,14 +1169,14 @@
                           'initial project plan',
              'audience': ['lead', 'pi', 'group_members'],
              'status': 'proposed',
-             'type': 'meeting'
+             'type': ALLOWED_MILESTONE_TYPES,
```
Comment (sbillinge) [inline regolith/schemas.py:1172]: this is not right here.  see if you can figure out the logic.....  What is the EXEMPLAR doing and what is the SCHEMA doing?  How are they doing it?

Comment (jc-umana) [reply regolith/schemas.py:1172]: If I'm being honest, from the file, I have no idea what the difference between the exemplar and the schema is or if they are necessarily separate.

Comment (sbillinge) [reply regolith/schemas.py:1172]: Thanks for the honesty.   Maybe a good topic for a convo with one of the other group members?

**Pull request closed without merging**